hudi-sync/hudi-dla-sync/src/main/java/org/apache/hudi/dla/DLASyncConfig.java

```diff
@@ -68,6 +68,9 @@
   @Parameter(names = {"--help", "-h"}, help = true)
   public Boolean help = false;
 
+  @Parameter(names = {"--support-timestamp"}, description = "If true, converts int64(timestamp_micros) to timestamp type")
+  public Boolean supportTimestamp = false;
```

Comment:
I think we need to add this option into DataSourceOptions, DataSourceUtils, and HoodieSparkSqlWriter

something like?
"hoodie.datasource.hive_sync.support_timestamp"

Reply:
@bschell  thank you for pointing this out. We only use standalone mode, so I missed this DataSourceOptions. Fixed now. PTAL.